### DISCUSSION TITLE
Model properties validation error

### DISCUSSION THREAD

**Discussion by preciousaang** [livewire/livewire#5521] [1. Help] [answered]
I have a livewire component with a public Model property of $user with

```
class CompanyEditModal extends Component
{
    public User $user;

    protected function rules()
    {
        return [
            'user.name' => 'required',
            'user.email' => ['required', 'email', Rule::unique('users')->ignore($this->user->id)]
        ];
    }
    public function render()
    {
        return view('livewire.company.company-edit-modal');
    }

    public function editCompany()
    {
        $this->validate();
    }
}
```

When I run the validation function,  I get the error below:
![livewire-error](https://user-images.githubusercontent.com/11736422/213772249-5788c5be-d6a7-43ee-8777-fee9939bfabc.png)

**Comment by joshhanley** [comment, marked as answer]
@preciousaang yeah the issue is with your `unique` validation as when the validation is run, the field name is generally `property.key` so in your instance it is `user.email`. You need to pass `'email'` as the second parameter in the `unique` method, like this:

```php
Rule::unique('users', 'email')->ignore($this->user->id)
```

Hope this helps!
